lib/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_profile/fia2milyas_custom_widget/fia2milyas_profile_all_followers.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fia2milyas/fia2milyas_view/fia2milyas_profile/fia2milyas_custom_widget/fia2milyas_profile_follower.dart';
+
+class FiA2MIlyasProfileAllFollower extends StatelessWidget {
```

Comment:
There's no need to make this class, just use a row in which ever file it is required.
Make a separate class/widget when either you're repeating the code or the code is too long.